src/lib/components/filters/content.svelte

```diff
@@ -98,7 +102,10 @@
     // This Map is keyed by tags, and has a query as the value
     function addFilter() {
         if (column && operator) {
```

Comment:
To avoid nested ifs, maybe we can do an early return here? E.g.

```suggestion
        if (!column || !operator) return
```